dts/bindings/spi/nordic,nrf-spi-common.yaml

```diff
@@ -15,14 +15,21 @@ properties:
     sck-pin:
       type: int
       required: true
-      description: SCK pin
+      description: |
+        SCK pin number. The value contains a GPIO pin in bits 0:4,
```

Comment:
Maybe we could follow the terminology from PS of our SoCs (see [SPI.PSEL.SCK in nRF52840](https://infocenter.nordicsemi.com/topic/ps_nrf52840/spi.html?cp=4_0_0_5_23_1#register.PSEL.SCK)) and use here "SCK pin selection. The value contains the pin number in bits ...".